src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -427,12 +427,6 @@ const NewWorkspaceModal = withDisplayName(
         const workspacePoliciesProps: WorkspacePoliciesProps = {
           workspace: cloneWorkspace,
           billingProject: selectedBillingProject,
-          title: 'Security, controls, and workspace information',
-          policiesLabel: 'Workspace will inherit data protection policies',
-          policiesLink: LinkWithPopout({
```

Comment:
The link is now always rendered, the label is no longer configurable, and we don't show a top-level title.
Before:
<img width="594" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/a63558a1-acff-4a76-bd1c-c82b4e3c074c">

After:
<img width="587" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/3114aafb-e62a-4c81-a4d0-da4adefb32e5">